packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -1070,3 +1071,75 @@ export const getWfaInterestFirstPerson = (
     `Invalid WFA interest '${wfaInterest}'`,
   );
 };
+
+const candidateFacingScreeningStage = defineMessages({
+  [ScreeningStage.NewApplication]: {
```

Comment:
When is this supposed to be implemented/used? I see it in the parent issue https://github.com/GCTC-NTGC/gc-digital-talent/issues/14389
But I don't think it is mentioned in the child issues

<img width="705" height="49" alt="Image" src="https://github.com/user-attachments/assets/36d75531-691b-42df-b505-70cfe4d37e1b" />

Reply:
Yeah, I added code to keep them insycn so we could avoid breaking changes for now. I tihnk we could do it as some computed field at the same time as the status changes? :thinking: 

- `pool_candidate_status` -> `application_status`
- `status` -> Computed using `application_status`, `screening_stage`, `placement_type`, etc.